test/vv/lib/python2.7/site-packages/plotly/api/v2/__init__.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import absolute_import
+
+from plotly.api.v2 import (dashboards, files, folders, grids, images, plot_schema,
```

Comment:
What is the point of this file? Is this a file for the purpose of testing out our v2 stuff?